package protoFile.shop.buyItemInMall214;
message buyItemInMallRequest{
     required int32 id = 1;//购买者角色Id
     required int32 itemTemplateId = 2;//物品的模板id
     required int32 count = 3;//商品的数量
     required int32 priceType=4;//1钻石 2绑定钻石
     required int32 buyType=5;//购买类型 0购买1赠送
     optional string presentName=6;//赠送的名称
}

message buyItemInMallResponse{
     required bool result = 1;
     optional string message = 2;
     optional ResponseData data=3;
}

message ResponseData{
     optional bool recharge =1;//是否需要提示充值
     optional int32 priceType= 20;// 1为钻石价格 2为绑定钻石价格  
     required int32 itemTemplateId = 3;//物品的模板id
     required int32 count = 4;//商品的数量
     required int32 buyType=5;//购买类型 0购买1赠送
     optional string presentName=6;//赠送的名称
} 